package.json

```diff
@@ -73,6 +73,7 @@
     "babel-loader": "8.0.6",
     "babel-plugin-module-resolver": "3.2.0",
     "babel-plugin-typescript-to-proptypes": "0.17.1",
+    "babel-preset-react-app": "9.1.1",
```

Comment:
Why this change?

Reply:
Oh! My mistake. I will remove it from this PR since it will be needed for adding `addons-doc` with typescript instead.

https://www.npmjs.com/package/@storybook/addon-docs

Good catch! 🧐